modules/serverless-kafka-source-autoscale-keda.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * eventing/event-sources/serverless-kafka-developer-source.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-source-autoscale-keda_{context}"]
+= Configuring KEDA autoscaling for KafkaSource
+
+You can configure Knative Eventing sources for Apache Kafka (KafkaSource) to be autoscaled using the Custom Metrics Autoscaler (KEDA) operator.
```

Comment:
Just want to check, does this operator need to be capitalized, like Operator? I know a lot of operators usually are capitalized O in the doc.

Reply:
Thanks! It does, and I expanded the description to be more precise: `the Custom Metrics Autoscaler Operator, which is based on the Kubernetes Event Driven Autoscaler (KEDA)`.